examples/temporal/supersample.glsl

```diff
@@ -0,0 +1,69 @@
+varying vec2 FragTexCoord0;
+uniform vec2 RenderSize;
+uniform sampler2D Texture0;
+uniform sampler2D Texture1;
+uniform vec4 halton;
```

Comment:
probably it would be useful to comments what's inside halton vec4 it's not obvious when reading the glsl code